app/views/users/show.html.erb

```diff
@@ -1,2 +1,23 @@
-<h1>Here you will find a specific user. The page will display user name, number of posts, bio and a summary of the posts</h1>
+<div class="user-details">
+  <div class="user-container">
+    <div class="user-img black-border">
+      <%= image_tag("icon.png") %>
+    </div>
```

Comment:
- [ ] Let's make use of the `user.photo` that was added when creating a user. This is a requirement and it's better to use it since it exists in the database instead of having the icon above for every user created.